includes/class-sensei-course.php

```diff
@@ -3912,6 +3912,44 @@ public function get_days_to_completion_total() {
 
 		return (int) $days_to_completion;
 	}
+
+	/**
+	 * Get average days to completion for all courses filtered by courses.
+	 *
+	 * @since 4.5.0
+	 * @access public
+	 *
+	 * @param array $courses_ids Courses ids to filter by.
+	 * @return int Total days to completion, rounded to the highest integer.
+	 */
+	public function get_days_to_completion_total_filter_courses( array $courses_ids ) : int {
```

Comment:
[The original method was changed recently](https://github.com/Automattic/sensei/pull/5097/files).

Reply:
@m1r0  @merkushin 
So I did a couple of things to address this and the comment beneath
1. Is I deprecated the original method 
2. I've added the private method with a mandatory parameter to class-sensei-reports-overview-list-table-courses.
3. This new method is not TOTALS but AVG to reflect the changes in the PR @merkushin  sent above
Here is the PR  
https://github.com/Automattic/sensei/pull/5091/commits/97de2d95f9866e518fc32d649105b2be900bf483
Here are the tests that I also moved and I applied the changes for average value: 
https://github.com/Automattic/sensei/pull/5091/commits/81ea9655a88f663710f4c3857ce45eed2ea73a6c